doc/index.rst

```diff
@@ -14,11 +14,10 @@
    :caption: Tutorials
 
    tutorials/01_getting_started/01_getting_started
-   tutorlals/03_online_analysis/03_online_analysis
-   tutorials/05_starting_ray/05_starting_ray
    tutorials/using_clients
    tutorials/lattice_boltz_analysis
    tutorials/inference
+   tutorials/05_starting_ray/05_starting_ray_builtin
```

Comment:
I see we are mixing numbered and not-numbered tutorials, like `01_getting_started` and `inference`. Would it make sense to simply switch to not-numbered ones? I'm concerned the mix could look confusing.

Reply:
The contents of the tutorials are number (that is sections in `using_clients` are labeled `2.x`). That said, I do like keeping the naming uniformity for all of the tutorial files. Adding numbers to the rst files

Reply:
Fixed in 43536be